.github/workflows/portal-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Portal CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+        architecture: 'x64'
+
+    - uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('context/requirements.txt') }}
```

Comment:
I have this memory of reading that caching pip in CI doesn't help much... There isn't that much repeated computation that's being saved... but maybe the github cache is significantly closer or faster than pypi?

Reply:
I just did this to mirror what we have in travis, I can remove it for simplicity?

Reply:
Not a strong preference... if it's in, and it works, fine to leave it, I think.